src/services/tx/tx-sender/__tests__/sdk.test.ts

```diff
@@ -0,0 +1,21 @@
+import { _getSupportedSigningMethods } from '@/services/tx/tx-sender/sdk'
+
+describe('getSupportedSigningMethods', () => {
+  it('should return `eth_signTypedData` for older Safes', () => {
+    const result = _getSupportedSigningMethods('1.0.0')
+
+    expect(result).toEqual(['eth_signTypedData'])
+  })
+
+  it('should return `eth_signTypedData` for invalid Safes', () => {
```

Comment:
hmm I'm curious why... does `null` really support `eth_signTypedData`?

Reply:
The backend can return `SafeInfo['version']` as `null` when it's an unsupported contract. Can you think of a better way to handle this, or should I adjust the test description?

Reply:
Ahh, then it makes sense. A better description (what you've just said about the backend and unsupported contract) would be nice 🙏